modules/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovySecurityTests.java

```diff
@@ -123,6 +123,12 @@ public void testEvilGroovyScripts() throws Exception {
         }
     }
 
+    public void testGroovyScriptsThatThrowErrors() throws Exception {
+        assertFailure("assert false, \"msg\";", AssertionError.class);
+        assertFailure("def foo=false; assert foo;", AssertionError.class);
+        assertFailure("def bar=false; assert bar, \"msg2\"", NoClassDefFoundError.class);
```

Comment:
I guess this is because we are not granting the class permission for `StringBuffer` but I think we should?